src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -275,7 +295,13 @@ def helm_install_release(chart_path, subscription_id, kubernetes_distro, kuberne
                         "--set", "global.onboardingPrivateKey={}".format(private_key_pem),
                         "--set", "systemDefaultValues.spnOnboarding=false",
                         "--set", "global.azureEnvironment={}".format(cloud_name),
+                        "--set", "systemDefaultValues.extensionoperator.enabled=true",
+                        "--set", "systemDefaultValues.clusterconnect-agent.enabled=true",
```

Comment:
This would be true by default in helm and isnt needed here

Reply:
I had added this after Atharva said, after discussion with kruupesh and shashank, we are keeping default to false in helm chart. If something else is decided, @athavamulmuley, can you take care of this now accordingly?

Reply:
I have removed for extensionoperator. We would still need for clusterconnect as it is disabled in charts by default.